Jetcaster/core/src/main/java/com/example/jetcaster/core/data/database/dao/EpisodesDao.kt

```diff
@@ -36,6 +36,15 @@ abstract class EpisodesDao : BaseDao<Episode> {
     )
     abstract fun episode(uri: String): Flow<Episode>
 
+    @Query(
+        """
+        SELECT episodes.* FROM episodes
+        INNER JOIN podcasts ON episodes.podcast_uri = podcasts.uri
+        WHERE episodes.uri = :uri
+        """
+    )
+    abstract fun episodeAndPodcast(uri: String): Flow<EpisodeToPodcast>
```

Comment:
This is not a blocker, but we could clarify what the uri parameter points at by changing the parameter name to podcastUri

Reply:
This is actually supposed to be `episodeUri`. But case in point, the variable name could be more descriptive. Will update.